src/scripts/components/VideoOverlay.js

```diff
@@ -247,28 +244,28 @@ class VideoOverlay extends Component<Props> {
             showShareModal={showShareModal}
           >
             {isEmbed && (
-              <ProfileButtonWrapper>
-                <IconButton
-                  color={
-                    activePlugin === PLAYER_PLUGIN.WALLET ? Colors.purple : ''
-                  }
-                  data-test-id="wallet-info-button"
-                  icon="/assets/img/profile.svg"
-                  onClick={(e: Object) => {
-                    e.stopPropagation()
-                    toggleActivePlugin(PLAYER_PLUGIN.WALLET)
-                  }}
-                />
+              <ProfileButtonWrapper
+                warn={activePlugin !== PLAYER_PLUGIN.WALLET}
+                accent={activePlugin === PLAYER_PLUGIN.WALLET}
+                data-test-id="wallet-info-button"
+                onClick={(e: Object) => {
+                  e.stopPropagation()
+                  toggleActivePlugin(PLAYER_PLUGIN.WALLET)
+                  console.log(e)
+                }}
+              >
+                <SVGIcon icon="icon-profile" />
               </ProfileButtonWrapper>
             )}
             <ShareButton
+              warn
```

Comment:
I don't think the `ShareButton` ever looks like a warning. I do think we need to rename this prop

Reply:
This is pretty related to the style. Perhaps put a "Style" in the prop name `warnStyle`.